02/app.js

```diff
@@ -1,28 +1,41 @@
-document.addEventListener('DOMContentLoaded', init);
+document.addEventListener("DOMContentLoaded", init);
 
 function init() {
-    const clickEl = document.querySelector('.error--click');
-    const enterEl = document.querySelector('.error--enter');
+  const clickEl = document.querySelector(".error--click");
+  const enterEl = document.querySelector(".error--enter");
 
-    setRandomPosition(clickEl);
-    setRandomPosition(enterEl);
-
-    initEventWithError(clickEl, 'click', new RangeError('Błąd zakresu!'));
-    initEventWithError(enterEl, 'mouseenter', new TypeError('Błąd typu!'));
+  setRandomPosition(clickEl);
+  setRandomPosition(enterEl);
 
+  initEventWithError(clickEl, "click", new RangeError("Błąd zakresu!"));
+  initEventWithError(enterEl, "mouseenter", new TypeError("Błąd typu!"));
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
+  element.style.top = Math.random() * 600 + "px";
+  element.style.left = Math.random() * 800 + "px";
 
-    if(error) {
-        throw error;
+  try {
+    if (error) {
+      throw error;
     }
+  } catch (e) {
+    const alertEl = document.querySelector(".alert--hidden");
+    alertEl.style.display = "block";
+    const pEl = document.querySelector(".alert__message");
+    pEl.textContent = e.message;
+  }
```

Comment:
Zwróć uwagę, że tutaj rzucamy błędem i od razu go przechwytujemy. Skoro tak to nie łatwiej zrobić normalny `if`? 
W rzucaniu błędami (wyjątkami) chodzi o to, aby  rzucić błędem i przechwycić go w innym miejscu.

Postaraj się poprawić :)